xinference/model/audio/model_spec_modelscope.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "model_name": "whisper-small.en",
+        "model_family": "whisper",
+        "model_id": "iic/speech_whisper-small_asr_english",
```

Comment:
Is this model identical to whisper-small.en? It looks not.

Reply:
Actually, I'm not very sure about this whisper-small.en from modelscope as shown in the picture.
![image](https://github.com/xorbitsai/inference/assets/112739514/119707a0-8007-4578-a590-38d5f3de4425)
If not, I will delete the corresponding JSON content.

Reply:
https://modelscope.cn/models/iic/speech_whisper-small_asr_english/summary

Reply:
You can see the description, it's different I think.

Reply:
I agree with your opinion and will delete the JSON file of this model.